packages/api-bindings/package.json

```diff
@@ -74,7 +74,7 @@
     "typescript": "5.2.2"
   },
   "peerDependencies": {
-    "@apollo/client": "~3.8.7",
+    "@apollo/client": "^3.8.10",
```

Comment:
is this version number correct?

Reply:
Not affecting the final result, but prefer to fix it. Good catch.